tools/glitchtip_access_revalidation.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from pathlib import Path
+
+import click
+
+from reconcile import mr_client_gateway
+from reconcile.cli import (
+    config_file,
+    dry_run,
+    gitlab_project_id,
+    log_level,
+)
+from reconcile.gql_definitions.glitchtip.glitchtip_project import (
+    query as glitchtip_project_query,
+)
+from reconcile.utils import gql
+from reconcile.utils.mr.labels import AUTO_MERGE
+from reconcile.utils.mr.notificator import (
+    CreateAppInterfaceNotificator,
+    Notification,
+)
+from reconcile.utils.runtime.environment import init_env
+
+EMAIL_BODY = """Hello App-Interface service owner,
+
+Access to all Glitchtip organizations and projects must be revalidated regularly.
+This ensures that the access is still valid and is needed to safeguard against unauthorized access.
+Please review, within one week, all your App-Interface roles referencing Glitchtip
+organizations (https://gitlab.cee.redhat.com/search?search=glitchtip_roles%3A&nav_source=navbar&project_id=13582&group_id=5301&search_code=true&repository_ref=master).
+
+If you have questions about this, please post a question in the #sd-app-sre Slack channel.
+
+Thank you,
+The SRE team
+"""
+
+
+@click.command()
+@config_file
+@dry_run
+@log_level
+@gitlab_project_id
+@click.option(
+    "--glitchtip-access-revalidation-workbook-path",
```

Comment:
What's this option for? In `main` this looks as the mail file that will be created, which is different to what this shows 🤔

Reply:
stupid copy & paste typo. fixed